site/content/docs/5.2/components/carousel.md

```diff
@@ -155,20 +155,23 @@ Add captions to your slides easily with the `.carousel-caption` element within a
       <div class="carousel-caption d-none d-md-block">
         <h5 class="mb-0">First slide label</h5>
         <p class="mb-0">Some representative placeholder content for the first slide.</p>
+        <a href="#" title="Example link #1">Example link #1</a>
```

Comment:
Since there's no dedicated CSS to render those links and since apparently there's no guidelines in the DSM, I'm not sure we need to add this in our docs.

Reply:
Yeah, I'll remove them from the PR since it was only a proposal of backport from v4.